### ISSUE TITLE
Small number of matches in search

### ISSUE THREAD

**Issue by ViolaChenYT** [sourmash-bio/sourmash#2168]
Hi,
Thanks for the great software!
I have been trying to query E_coli_iai39 fasta sequence (signature prepared using sourmash) against GTDB representative genomes (I've tried the pre-built genome on the website as well as building from the original GTDB data)

I have been getting only 6 matches despite using the -n option in attempt to get more. All the matches have pretty high ANI (>94%) and I would like to get more species that have slightly lower ANI (like around 90%). I wonder if there is a way to do so?

Thanks a lot!

P.S. attaching the output below
```
$ sourmash search Escherichia_coli_iai39.fasta.sig prebuiltrep.sbt.gz -n 10 -o using_prebuilt.csv
== This is sourmash version 4.4.2. ==
== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==

select query k=31 automatically.
loaded query: ../refs/Escherichia_coli_iai39... (k=31, DNA)
loaded 1 databases.

6 matches:
similarity   match
----------   -----
 32.6%       GCF_003697165.2 Escherichia coli DSM 30083 = JCM 1649 = A...
 12.5%       GCF_002965065.1 Escherichia sp. MOD1-EC7003 strain=MOD1-E...
 10.6%       GCF_004211955.1 Escherichia sp. E1V33 strain=E1V33, ASM42...
  9.8%       GCF_902498915.1 Escherichia ruysiae, OPT1704
  9.6%       GCF_005843885.1 Escherichia sp. E4742 strain=E4742, ASM58...
  9.6%       GCF_000026225.1 Escherichia fergusonii ATCC 35469 strain=...
```

**Comment by ctb** [comment]
hi! try `--threshold` --

```
sourmash search --threshold=0 ...
```

(I've always found `-n` confusing despite being the one who implemented it. Maybe I should just remove it...)

**Comment by ViolaChenYT** [comment]
Thank you so much for your prompt reply! Issue solved :)
(Perhaps a less confusing way of implementing `-n` is to do `--threshold=0` by default and take the top `N` results, but I guess that could slow it down :P )

**Comment by ctb** [comment]
> Thank you so much for your prompt reply! Issue solved :) (Perhaps a less confusing way of implementing `-n` is to do `--threshold=0` by default and take the top `N` results, but I guess that could slow it down :P )

that is exactly the reasoning behind not having a lower `--threshold` by default! not sure it matters any more, because we've gotten much faster...

but, yes, I think we need to reconsider how we do `-n`... I'll create a new issue. Thank you so much for bringing this up!

**Comment by ctb** [comment]
(note, this is at least addressed in https://github.com/sourmash-bio/sourmash/pull/2222, which will be released in sourmash v4.5.0)
